### PR TITLE
Feature preview testing/small styling touchup

### PR DESCRIPTION
**Pull request by lbmerchant93**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Add testing for:
error handling
loading
details on about page
details on picture of the day page
(previous and next day sections are there)

## *Instructions for reviewer:*
*Where should the reviewer start?*
picture-of-the-day-spec.js

*How should this be manually tested?*
run tests in cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: cypress/fixtures/spec_date_data.json [hunk 1]
```diff
@@ -12,6 +12,8 @@
   
   "title":"This is what peak performance looks like.",
 
+  "copyright": "Richard, Cole, Lucas",
```
Comment (richardltyler) [inline cypress/fixtures/spec_date_data.json:15]: Good eye for the `copyright` in the data

File: cypress/integration/devs-spec.js [hunk 2]
```diff
@@ -11,25 +11,27 @@ describe("Devs Component", () => {
...
+    cy.get('#richard .dev-gh').should('have.attr', 'href', 'https://github.com/richardltyler');
   })
 
   it('should navigate to each dev\'s linkedin', () => {
-    cy.get('article:first a:last').should('have.attr', 'href', 'https://www.linkedin.com/in/lucas-merchant93/');
-
-    cy.get('article a').eq(3).should('have.attr', 'href', 'https://www.linkedin.com/in/colefiscus/');
-
-    cy.get('article a').eq(5).should('have.attr', 'href', 'https://www.linkedin.com/in/richardltyler/');
+    cy.get('#lucas .dev-li').should('have.attr', 'href', 'https://www.linkedin.com/in/lucas-merchant93/');
```
Comment (richardltyler) [inline cypress/integration/devs-spec.js:33]: I like using the id and the class here. Very specific.

File: cypress/integration/picture-of-the-day-spec.js [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
 describe("Pic of the Day component", () => {
+  const today = moment().format('YYYY-MM-DD')
 
   beforeEach(() => {
+
     cy.fixture('potd_data.json')
-      .then((response) => {
-        cy.intercept('GET', 'https://api.nasa.gov/planetary/apod?api_key=j9VLjGbdXCRXtf61nCle9dLGtNzWVnNqUM1BNV86', {
+      .then(() => {
+        cy.intercept('GET', `https://api.nasa.gov/planetary/apod?date=${today}&api_key=j9VLjGbdXCRXtf61nCle9dLGtNzWVnNqUM1BNV86`, {
```
Comment (richardltyler) [inline cypress/integration/picture-of-the-day-spec.js:10]: Smart move bringing in a dynamic date

File: cypress/integration/picture-of-the-day-spec.js [hunk 4]
```diff
@@ -30,18 +42,105 @@ describe("Pic of the Day component", () => {
...
+  beforeEach(() => {
+    
+    cy.intercept('GET', `https://api.nasa.gov/planetary/apod?date=${today}&api_key=j9VLjGbdXCRXtf61nCle9dLGtNzWVnNqUM1BNV86`, {
+      statusCode: 500
+    })
+    cy.visit("http://localhost:8080")
+      .get(".enter-site").click() 
+  })
+  it('Should display the error number and a message when something goes wrong with the api call to retrieve the photo of the day information is not ok', () => {
+    cy.get('main').contains("500 error. Sorry! Something went wrong! Try again later or go to Contact Us to contact the developers with questions!")
```
Comment (colefiscus) [inline cypress/integration/picture-of-the-day-spec.js:124]: Thank you for testing the errors!

File: src/views/Footer.vue [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<template>
```
Comment (richardltyler) [inline src/views/Footer.vue:1]: I like having the Footer as it's own component

File: src/views/About.vue [hunk 6]
```diff
@@ -55,6 +58,11 @@
     color: #49A8C6;
   }
 
+  .devs-link {
+    text-align: center;
+    padding: 0 50px;
```
Comment (colefiscus) [inline src/views/About.vue:63]: And thanks for the minor fixes!

**Review by richardltyler — APPROVED** [review]
Looks great!

**Review by colefiscus — APPROVED** [review]
Excellent work with extra thorough testing.

**Pull request merged**